lib/matplotlib/tests/test_offsetbox.py

```diff
@@ -284,31 +285,24 @@ def test_annotationbbox_extents():
                          arrowprops=dict(arrowstyle="->"))
     ax.add_artist(ab6)
 
-    fig.canvas.draw()
     renderer = fig.canvas.get_renderer()
```

Comment:
What about dropping this line (and update below) to also test the renderer-fetching part?

Reply:
Good idea!  Done.